pages/storage_and_backup/object_storage/s3_asynchronous_replication/guide.en-gb.md

```diff
@@ -0,0 +1,407 @@
+| Destination             | A container for information about the replication destination and its configurations.                                    | Yes      |
+| DeleteMarkerReplication | Tells if delete operations should be replicated.                                                                         | Yes      |
+| Bucket                  | The destination bucket (to replicate to multiple destinations, you must create multiple replication rules).              | Yes      |
+| And                     | You can apply multiple selection criteria in the filter.                                                                 | No       |
+
+### Delete marker replication
+
+> [!warning]
+> **IMPORTANT**
+> 
+> If you specify a `Filter` in your replication configuration, you **must** also include a `DeleteMarkerReplication` element. If your `Filter` includes a `Tag` element, the `DeleteMarkerReplication` Status **must be set to _Disabled_**.
+>
+
+### Understanding delete markers
+
+When a delete object operation is performed on an object in a versioning-enabled bucket, it does not delete the object permanently but it creates a delete marker on the object. This delete marker becomes the latest and current version of the object with a new version ID.
+
+A delete marker has the following properties:
+
+- A key and version ID like any other object.
+- It does not have data associated with it, thus it does not retrieve anything from a `GET` request (you get a 404 error).
+- By default, it is not displayed in the Control Panel UI anymore.
+- The only operation you can use on a delete marker is `DELETE`, and only the bucket owner can issue such a request.
+
+To permanently delete an object, you have to specify the version ID in your `DELETE` object request.
+
+> [!warning]
+> By default, OVHcloud Object Storage does not replicate delete markers nor replicate the permanent deletion to destination buckets. This behavior protects your data from unauthorized or unintentional deletions.
+
+However, you can still replicate delete markers by adding the `DeleteMarkerReplication` element to your replication configuration rule. `DeleteMarkerReplication` specifies if delete markers should or should not be replicated (when versioning is enabled, a delete operation is performed on an object it does not actually delete the object but it flags it with a delete marker).
+
+```json
+{
+  "Role": "arn:aws:iam::<your_project_id>:role/s3-replication",
+  "Rules": [
+    {
+      ...
+      "DeleteMarkerReplication": {
+        "Status": "Enabled"|"Disabled"
+      }
+    }
+  ]
+}
+```
+
+### Checking the replication status
+
+The replication status can be used to determine the status of an object that is being replicated. When you request a source object (using `GET object`) or source object metadata (using `HEAD object`), OVHcloud Object Storage returns the replication status via the header `x-amz-replication-status`.
```

Comment:
feedbacks dev: supprimer la phrase "when you request.... replication-status" par "To get the replication status of an object, you can use the head-object command via the aws cli:
```bash
$ aws s3api head-object --bucket <source_bucket> --key <object_name>
{
   "LastMoodified" : "Fri, 15 Mar 2024 10:18:15 GMT",
   "ContentLength": 3481,
   "Etag": "\"417947d3634d4645e05ca9e875f5b202\"",
   "VersionId": "17104978950.04081",
   "ContentType": "binary/octet-stream",
   "Metadata": { },
   "StorageClass": "STANDARD",
   "ReplicationStatus": "COMPLETED"
}
```